src/prefect_server/graphql/flows.py

```diff
@@ -22,7 +22,7 @@ async def resolve_create_flow_from_compressed_string(
 @mutation.field("create_flow")
 async def resolve_create_flow(obj: Any, info: GraphQLResolveInfo, input: dict) -> dict:
     serialized_flow = input["serialized_flow"]
-    project_id = input["project_id"]
+    project_id = input.get("tenant_id", None)
```

Comment:
```suggestion
    project_id = input.get("project_id", None)
```

Reply:
> Is project_id already nullable in Server or is this PR missing a migration?

This is missing a migration -- I wrote the migration in Cloud and had to use this branch to get my Cloud tests going. I'll add the migration here as well and cleanup the Server tests soon.